City-of-Helsinki/helsinki-paatokset
# PP-255: Drupal core and modules updates, Search app update

**To test:**
- Checkout branch, run `make fresh` (take a database backup first)
- Run `make ahjo-migrations` to get some decisions content (if you don't have any)
- Make sure you have indexed content in the decisions index: https://helsinki-paatokset.docker.so/fi/admin/config/search/search-api/index/decisions
- Follow these test instructions on the decisions search page: https://github.com/City-of-Helsinki/paatokset-search/pull/22
- Check that there are no pending security updates: https://helsinki-paatokset.docker.so/en/admin/reports/updates
- Click around the site and check that decision and organization related content works as it should

**Updates:**
```
- Upgrading commerceguys/addressing (v1.3.0 => v1.4.1)
- Upgrading composer/ca-bundle (1.3.2 => 1.3.3)
- Upgrading composer/composer (2.2.14 => 2.2.17)
- Upgrading composer/semver (3.2.6 => 3.3.2)
- Upgrading consolidation/annotated-command (4.5.4 => 4.5.6)
- Upgrading doctrine/annotations (1.13.2 => 1.13.3)
- Upgrading doctrine/lexer (1.2.1 => 1.2.3)
- Upgrading doctrine/reflection (1.2.2 => 1.2.3)
- Upgrading drupal/address (1.10.0 => 1.11.0)
- Upgrading drupal/config_filter (2.3.0 => 2.4.0)
- Upgrading drupal/config_rewrite (1.4.0 => 1.5.0)
- Upgrading drupal/core (9.3.16 => 9.4.5)
- Upgrading drupal/core-composer-scaffold (9.3.16 => 9.4.5)
- Upgrading drupal/core-dev (9.3.16 => 9.4.5)
- Upgrading drupal/core-recommended (9.3.16 => 9.4.5)
- Upgrading drupal/ctools (3.7.0 => 4.0.1)
- Upgrading drupal/default_content (2.0.0-alpha1 => 2.0.0-alpha2)
- Upgrading drupal/easy_breadcrumb (2.0.2 => 2.0.3)
- Upgrading drupal/editoria11y (1.0.13 => 1.0.14)
- Upgrading drupal/entity_browser (2.6.0 => 2.8.0)
- Upgrading drupal/externalauth (2.0.0 => 2.0.2)
- Upgrading drupal/filelog (2.1.0 => 2.1.1)
- Upgrading drupal/gin (3.0.0-alpha37 => 3.0.0-beta5)
- Upgrading drupal/gin_toolbar (1.0.0-beta20 => 1.0.0-beta22)
- Upgrading drupal/helfi_tpr (2.0.11 => 2.1.1)
- Upgrading drupal/helfi_tunnistamo (2.0.6 => 2.0.7)
- Upgrading drupal/matomo (1.18.0 => 1.19.0)
- Upgrading drupal/matomo_reports (1.1.0 => 1.2.0)
- Upgrading drupal/metatag (1.19.0 => 1.21.0)
- Upgrading drupal/migrate_plus (5.2.0 => 5.3.0)
- Upgrading drupal/oembed_providers (2.0.2 => 2.1.0)
- Upgrading drupal/paragraphs_asymmetric_translation_widgets (1.0.0-beta4 => 1.0.0)
- Upgrading drupal/pathauto (1.9.0 => 1.11.0)
- Upgrading drupal/search_api (1.23.0 => 1.25.0)
- Upgrading drupal/simple_sitemap (4.1.1 => 4.1.2)
- Upgrading drupal/token (1.10.0 => 1.11.0)
- Upgrading drupal/views_bulk_operations (4.1.2 => 4.1.4)
- Upgrading egulias/email-validator (3.1.2 => 3.2.1)
- Upgrading guzzlehttp/guzzle (6.5.7 => 6.5.8)
- Upgrading guzzlehttp/psr7 (1.8.5 => 1.9.0)
- Upgrading josdejong/jsoneditor (v5.29.1 v5.29.1 => v5.29.1 )
- Upgrading laminas/laminas-diactoros (2.8.0 => 2.11.3)
- Upgrading laminas/laminas-feed (2.15.0 => 2.17.0)
- Upgrading laminas/laminas-stdlib (3.6.1 => 3.7.1)
- Upgrading league/uri (6.5.0 => 6.7.1)
- Upgrading makinacorpus/php-lucene (1.0.4 => 1.1.0)
- Upgrading mikey179/vfsstream (v1.6.10 => v1.6.11)
- Upgrading paatokset/paatokset_search (1.0.4 => 1.0.5)
- Upgrading phpstan/phpdoc-parser (1.6.2 => 1.7.0)
- Upgrading psr/container (1.1.1 => 1.1.2)
- Upgrading ruflin/elastica (dev-master 72a4598 => dev-master 8fe6176)
- Upgrading sirbrillig/phpcs-variable-analysis (v2.11.3 => v2.11.4)
- Upgrading squizlabs/php_codesniffer (3.6.2 => 3.7.1)
- Upgrading symfony/browser-kit (v4.4.37 => v4.4.44)
- Upgrading symfony/console (v4.4.34 => v4.4.44)
- Upgrading symfony/css-selector (v4.4.37 => v4.4.44)
- Upgrading symfony/debug (v4.4.31 => v4.4.44)
- Upgrading symfony/dependency-injection (v4.4.34 => v4.4.44)
- Upgrading symfony/deprecation-contracts (v2.5.0 => v2.5.2)
- Upgrading symfony/dom-crawler (v4.4.42 => v4.4.44)
- Upgrading symfony/error-handler (v4.4.34 => v4.4.44)
- Upgrading symfony/event-dispatcher (v4.4.34 => v4.4.44)
- Upgrading symfony/event-dispatcher-contracts (v1.1.11 => v1.1.13)
- Upgrading symfony/finder (v4.4.41 => v4.4.44)
- Upgrading symfony/http-client-contracts (v2.5.0 => v2.5.2)
- Upgrading symfony/http-foundation (v4.4.34 => v4.4.44)
- Upgrading symfony/http-kernel (v4.4.35 => v4.4.44)
- Upgrading symfony/mime (v5.4.0 => v5.4.11)
- Upgrading symfony/phpunit-bridge (v5.4.8 => v5.4.11)
- Upgrading symfony/polyfill-ctype (v1.23.0 => v1.25.0)
- Upgrading symfony/polyfill-iconv (v1.23.0 => v1.25.0)
- Upgrading symfony/polyfill-intl-idn (v1.23.0 => v1.25.0)
- Upgrading symfony/polyfill-intl-normalizer (v1.23.0 => v1.25.0)
- Upgrading symfony/polyfill-mbstring (v1.23.1 => v1.25.0)
- Upgrading symfony/polyfill-php80 (v1.23.1 => v1.25.0)
- Upgrading symfony/process (v4.4.35 => v4.4.44)
- Upgrading symfony/routing (v4.4.34 => v4.4.44)
- Upgrading symfony/serializer (v4.4.35 => v4.4.44)
- Upgrading symfony/service-contracts (v2.5.0 => v2.5.2)
- Upgrading symfony/translation (v4.4.34 => v4.4.44)
- Upgrading symfony/translation-contracts (v2.5.0 => v2.5.2)
- Upgrading symfony/validator (v4.4.35 => v4.4.44)
- Upgrading symfony/var-dumper (v5.4.0 => v5.4.11)
- Upgrading symfony/yaml (v4.4.34 => v4.4.44)
- Upgrading twig/twig (v2.14.11 => v2.15.1)
```
